storage/store.go

```diff
@@ -901,8 +932,8 @@ func (s *Store) processRaft(r raftInterface, closer chan struct{}) {
 			r, ok := s.ranges[raftCmd.cmd.RaftID]
 			s.mu.Unlock()
 			if !ok {
-				log.Errorf("got committed raft command for %d but have no range with that ID",
-					raftCmd.cmd.RaftID)
+				log.Errorf("got committed raft command for %d but have no range with that ID: %+v",
```

Comment:
did you see this error during testing? We should also make sure that Multiraft handles commands to nonexisting ranges but that's outside the scope of this PR.